tools/flect/initializer.go

```diff
@@ -0,0 +1,57 @@
+package flect
+
+import (
+	"context"
+	"fmt"
+	"os"
+)
+
+type Initializer struct{}
+
+func (i Initializer) Name() string {
+	return "flect/initializer"
+}
+
+func (i *Initializer) Initialize(ctx context.Context, root string, args []string) error {
+
+	root = root + "/inflections.yml"
+
+	content := `
+	{
+	  "singular": "plural"
+	}
+	`
+
+	if _, err := os.Stat(root); err == nil {
+
+		fmt.Println("inflections.yml file already exist")
+		return nil
+
+	} else if os.IsNotExist(err) {
+
+		// create file if it does not exist
+		file, err := os.Create(root)
+
+		if err != nil {
+			return (err)
+		}
+
+		_, err = os.OpenFile(root, os.O_RDWR, 0644)
+		if err != nil {
+			return (err)
+		}
+
+		_, err = file.WriteString(content)
+		if err != nil {
+			return (err)
+		}
+
+		file.Close()
+
+		return nil
+
+	} else {
+		return err
+
+	}
```

Comment:
```suggestion
         _, err := os.Stat(root); 
        If err == nil {
                fmt.Println("inflections.yml file already exist")
  		return nil
        }
        
	if os.IsNotExist(err){
	   // create file if it does not exist
		file, err := os.Create(root)

		if err != nil {
			return (err)
		}

		_, err = os.OpenFile(root, os.O_RDWR, 0644)
		if err != nil {
			return (err)
		}

		_, err = file.WriteString(content)
		if err != nil {
			return (err)
		}

		file.Close()

		return nil
	} 
	
        return err
```

Reply:
@YanDeLeon see how this is simpler in terms of nesting and logical complexity.

Reply:
(Yet it covers the same cases)

Reply:
Got it. Thanks !